books/free-programming-books-id.md

```diff
@@ -83,6 +83,7 @@
 * [Ebook Belajar HTML Dan CSS Dasar](https://www.malasngoding.com/download-ebook-belajar-html-dan-css-dasar-gratis/)
 * [Tutorial Dasar CSS untuk Pemula](https://www.petanikode.com/tutorial/css/) - Ahmad Muhardian (Petani Kode) *(:construction: Dalam Proses)*
 * [Tutorial HTML untuk Pemula](https://www.petanikode.com/tutorial/html/) - Ahmad Muhardian (Petani Kode)
+* [Belajar HTML dari Nol: Pengenalan Dasar HTML untuk Pemula](https://www.petanikode.com/html-dasar/) - Ahmad Muhardian (Petani Kode)
```

Comment:
Sort the resource alphabetically.  Look at [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more information